charts/stable/firefly-iii/values.yaml

```
@@ -0,0 +1,462 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: fireflyiii/core
+  # -- image pull policy
+  pullPolicy: IfNotPresent
+  # -- image tag
+  tag: version-5.6.14
```

```suggestion
  # @default -- chart.appVersion
  tag:
```
These changes, coupled with the adjustment in chart.yaml will allow maintaing the image tag in one place not two